WebRoot/js/zimbraAdmin/domains/model/ZaDomain.js

```diff
@@ -751,6 +751,16 @@ function(tmpObj, newDomain) {
 		attr.setAttribute("n", ZaDomain.A_zimbraAdminConsoleCatchAllAddressEnabled);
 	}
 
+	if(tmpObj.attrs[ZaDomain.A_zimbraFeatureMailRecallEnabled]) {
+		attr = soapDoc.set("a", tmpObj.attrs[ZaDomain.A_zimbraFeatureMailRecallEnabled]);
+		attr.setAttribute("n", ZaDomain.A_zimbraFeatureMailRecallEnabled);
+	}
+
+	if(tmpObj.attrs[ZaDomain.A_zimbraFeatureMailRecallTime]) {
+		attr = soapDoc.set("a", tmpObj.attrs[ZaDomain.A_zimbraFeatureMailRecallTime]);
+		attr.setAttribute("n", ZaDomain.A_zimbraFeatureMailRecallTime);
+	}
+
```

Comment:
Can't we add them from an admin zimlet?

Reply:
We tried the same to achieve it with zimlet i.e. pushing it in the tmpObj but in order to update the payload we need to update the soapDoc present inside CreateMethod(which is not possible from zimlet as we are not able to update soapDoc)and invoke the CreateDomainResponse to send the updated field values.

Reply:
ok, I will check and get back to you

Reply:
@k-kato as discussed for this PR we are going ahead with the approach that globally defined extendedAttrs can be used by other Admin Extensions to add LDAP attributes in Future. 
Apart from this the createMethod for CreateDomain API needs to be optimised and tested thoroughly, which will be handled in https://jira.corp.synacor.com/browse/ZCS-13787.